cdap-runtime-ext-dataproc/src/main/java/co/cask/cdap/runtime/spi/provisioner/dataproc/DataProcProvisioner.java

```diff
@@ -187,6 +189,11 @@ public PollingStrategy getPollingStrategy(ProvisionerContext context, Cluster cl
     return strategy;
   }
 
+  @Override
+  public Set<String> getCapabilities() {
+    return ImmutableSet.of("gcp");
```

Comment:
nit: Collections.singleton()

It's better to use standard java classes when it is easy to do so, in case we ever want to remove guava for some reason.